hail/src/main/scala/is/hail/expr/ir/TableIR.scala

```diff
@@ -40,6 +41,30 @@ abstract sealed class TableIR extends BaseIR {
     fatal("tried to execute unexecutable IR:\n" + Pretty(this))
 
   override def copy(newChildren: IndexedSeq[BaseIR]): TableIR
+
+  def persist(storageLevel: StorageLevel): TableIR = {
+    val tv = Interpret(this)
+    TableLiteral(tv.persist(storageLevel))
+  }
+
+  def unpersist(): TableIR = {
+    val tv = Interpret(this)
+    TableLiteral(tv.unpersist())
+  }
+
+  def pyPersist(storageLevel: String): TableIR = {
```

Comment:
Can you help me understand the reason for these two methods? Why not just call the above two directly?

Reply:
1. `pyPersist` takes a String, not a StorageLevel (which we can't create in Python).

2. I've started prefixing Python entrypoints with `py` . This is mostly so it is easy to find dead code: if it isn't prefixed by `py` and it isn't used, it can be deleted.  If it is prefixed by `py`, you have to check the Python code.  I'm planning another PR that renames other entrypoints.